modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -996,6 +1007,26 @@ private void _validateName(
 		}
 	}
 
+	private void _validateRelationship(long objectFieldId, boolean required)
+		throws ObjectFieldRelationshipTypeException {
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipPersistence.fetchByObjectFieldId2(objectFieldId);
+
+		if ((objectRelationship != null) &&
```

Comment:
objectRelationship will always be different than null.

Reply:
And if we fetch with the objectFieldId that doesn't exist in objectRelationship?